site/content/downloads/_index.md

```diff
@@ -17,7 +17,6 @@
 # specific language governing permissions and limitations
 # under the License.
 #
-title: "Releases"
```

Comment:
The title creates too much foot prints on the page, remove it to make the page clean.